src/daemon/org.projectatomic.rpmostree1.conf

```diff
@@ -37,5 +37,9 @@
     <allow send_destination="org.projectatomic.rpmostree1"
            send_interface="org.projectatomic.rpmostree1.Sysroot"
            send_member="GetOS"/>
+
+    <allow send_destination="org.projectatomic.rpmostree1"
+           send_interface="org.projectatomic.rpmostree1.Sysroot"
+           send_member="RegisterClient"/>
```

Comment:
That makes sense, yeah. I saw the `at_console` attribute for this, but then found https://lintian.debian.org/tags/dbus-policy-at-console.html and https://bugs.freedesktop.org/show_bug.cgi?id=39611, which seems to discourage its use in favour of using polkit. Will try that out.